incubator/opa/templates/opa-rbac.yaml

```diff
@@ -0,0 +1,38 @@
+# Grant OPA/kube-mgmt read-only access to resources. This lets kube-mgmt
+# replicate resources into OPA so they can be used in policies.
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: {{ include "opa.fullname" . }}-viewer
+roleRef:
+  kind: ClusterRole
+  name: view
+  apiGroup: rbac.authorization.k8s.io
+subjects:
+- kind: Group
+  name: system:serviceaccounts:opa
```

Comment:
Do these service account groups references here and below break down if we install to a namespace other than `opa`?

Reply:
Hmmm.  I will have to see.

Reply:
Turns out I didn't notice this terrible implementation of RBAC from the examples.  I am going to refactor this to use a more standard rbac pattern where we create a service account and bind that to the roles and run the pod with that serviceaccount.